buildvertical.targets

```diff
@@ -1,20 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <UsingTask TaskName="FindBestConfiguration" AssemblyFile="$(CoreFxToolsTaskDir)CoreFx.Tools.dll"/>
-
-  <Import Project="buildvertical.props" />
-
   <PropertyGroup>
     <ImportedBuildVerticalTargets>true</ImportedBuildVerticalTargets>
   </PropertyGroup>
-  
+
+ <!-- Required for FindBestConfiguration task -->
+ <Import Project="$(MSBuildThisFileDirectory)src/Tools/GenerateProps/properties.props" />
```

Comment:
My impression was that the tasks would live in build tools and potentially even a default set of properties, but also allow repos to define their own.